10-query/01b-pattern-negation.md

```diff
@@ -266,15 +410,15 @@ then {
 ```
 
 Please note the explicit addition of the `node` types in the body of the rule. This is to to ensure the boundedness condition
-is satisfied as well as to maintain the expected meaning of `unreachable` relation.
-
-
+is satisfied as well as to maintain the expected meaning of the `unreachable` relation.
 
 ## Monotonicity of reasoning with negation
 
-In this subsection we will address an important property of reasoning being the monotonicity property.
-We say that reasoning is monotonic if previously derived facts remain true upon addition of new knowledge. Up to this point Graql reasoning was monotonic. However with
-the addition of pattern negation this property no longer holds. We can illustrate this fact in a popular example:
+In this subsection we address an important property of reasoning - the monotonicity property.
+We say that reasoning is monotonic if previously derived facts remain true upon addition of new knowledge. 
+Without employing pattern negation in rules, Graql reasoning is monotonic. When pattern negation is used in rules however, this property no longer holds.
+Preserving monotonicity property is not strictly necessary or required, however it is important to be aware of the consequences of non-monotonic reasoning 
```

Comment:
Either:
_Preserving the monotonicity property_
_Preserving monotonicity_
The second sounds better